src/atsu/src/com/FutureAirNavigationSystem.ts

```diff
@@ -0,0 +1,72 @@
+//  Copyright (c) 2022 FlyByWire Simulations
+//  SPDX-License-Identifier: GPL-3.0
+
+export enum FansMode {
+    FansNone,
+    FansA,
+    FansB
+}
+
+export class FutureAirNavigationSystem {
+    // contains all CPDLC callsigns that use FANS-B
```

Comment:
Can we specify the source here to help somebody wanting to update this in future?